.coderabbit.yml

```diff
@@ -0,0 +1,32 @@
+language: "en"
+early_access: false
+reviews:
+    request_changes_workflow: false
+    high_level_summary: true
+    poem: false
+    review_status: true
+    collapse_walkthrough: true
+    path_filters:
+        - "!api/"
+    path_instructions:
+        - path: "**/*.go"
+          instructions: "Review the Golang code for conformity with the Uber Golang style guide, highlighting any deviations."
+        - path: "tests/**/*"
+          instructions: |
+              "Assess the integration and e2e test code assessing sufficient code coverage for the changes associated in the pull request"
+        - path: "**/*_test.go"
+          instructions: |
+              "Assess the unit test code assessing sufficient code coverage for the changes associated in the pull request"
+        - path: "docs/**/*.md"
```

Comment:
Should be all markdown, modules docs and other are not under docs/

Reply:
changed